app/scripts/metamask-controller.js

```diff
@@ -1187,12 +1187,15 @@ export default class MetamaskController extends EventEmitter {
       },
       version,
       // account mgmt
-      getAccounts: async ({ origin }) => {
+      getAccounts: async ({ origin }, suppressUnauthorizedError = true) => {
```

Comment:
Since this method already takes an object, should `suppressUnauthorizedError` be put inside the object? That way if other arguments get added in the future we do not have to worry about supplying a value for `suppressUnauthorizedError`, etc.